orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/functions/UrlExpressionFunctionProvider.java

```diff
@@ -80,7 +77,11 @@ public Functions getFunctions() {
             new FunctionParameter(String.class, "value", "A String containing JSON text")),
         new FunctionDefinition(
             "readYaml",
-            "Parses YAML from a string to be accessed, parsed JSON can be accessed as an object",
+            "Parses YAML from a string to be accessed, parsed YAML can be accessed as an object",
+            new FunctionParameter(String.class, "value", "A String containing YAML text")),
+        new FunctionDefinition(
+            "readYamlAll",
```

Comment:
Prefer `readAllYaml` -- the name as it exists in the PR doesn't make a lot of sense.

Reply:
Fixed!